src/qlever/__init__.py

```diff
@@ -10,9 +11,12 @@ def snake_to_camel(str):
     return "".join([w.capitalize() for w in str.replace("-", "_").split("_")])
 
 
+# Get the name of the script (without the path and without the extension).
+script_name = Path(sys.argv[0]).stem
```

Comment:
Just add a variable `engine_name` here, which is just the `script_name` without the initial `q` and then capitalized by default. For `qlever`, it should be `QLever`. Also check the other engine names whether they make sense